algebra/group.lean

```diff
@@ -691,6 +691,14 @@ protected lemma is_conj (f : α → β) [is_group_hom f] {a b : α} : is_conj a
 lemma to_is_monoid_hom (f : α → β) [is_group_hom f] : is_monoid_hom f :=
 ⟨is_group_hom.one f, is_group_hom.mul f⟩
 
+@[to_additive is_add_group_hom.injective_iff]
+lemma injective_iff (f : α → β) [is_group_hom f] :
```

Comment:
This is very similar to `group_theory/subgroup.lean:lemma inj_iff_trivial_ker`.

Reply:
But you could use that to shorten the proof?

Reply:
This doesn't import `subgroup`

Reply:
You may want to do it the other way round, i.e. base `inj_iff_trivial_ker` on `injective_iff`.
I'm not sure if it exists, but you might want a rule a la `g = trivial α ↔ g <= trivial α` (which you could get from the lattice structure of the subgroups)